_posts/2019-03-20-tmm-post.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "Trump Money Maker: Timing the Market"
+date:   2019-03-20 10:39:11 -0500
+categories: blog
+---
+_Major lessons learned during the development of Trump Money Maker._
+[![Trump Money Maker automated teller gif stamping envelopes]({{ "/assets/teller.gif" | relative_url }}){:class="float-right-plz teller-img"}]({{ site.baseurl }}{% post_url 2019-03-20-tmm %})
+<!-- Ends the excerpt text, it includes the image -->
+
+
+<br>
+
+
+Imagine 6 bright, eager, and adventurous roommates. Sam, RJ, and I lived with 3 of our friends in an apartment we called, _The Mansion_. It was campaign season, early 2016, and we were very hot for politics and game dev. Inevitably, those passions bled together into TMM.
+
+## The Goods
+
+We talked about the game idea constantly. I created mockups. I even had an animation for a wall building game that was the original idea for TMM. We involved the whole house and did a lot of **prototyping, user research, and testing**. In that respect, the idea was built into something that I think is really fun.
+
+
+The dev cycle was constantly injected with **feedback** from everyone in the house. We had many [releases](https://github.com/SylverStudios/trump-money-maker/releases) which were hilarious, but deployable and testable. This turned out to be crucial. We may do a post on the development experience if there is interest, but I would say it was the most effective and collaborative project that we had done yet.
+
+
+The political theme led to a lot of **humor**, which has always been a big motivator for us, but the availability of content and sound bites really made that come to the forefront.
+
+## The Bads
+
+This was the first time we've ever had a **due Date** on a project! We hardly released it before the election, and it would have had more time in the spotlight had we released earlier than that. This does have some pros, like being ruthless in our prioritization, but that's a topic for another post
```

Comment:
```suggestion
This was the first time we've ever had a **due date** on a project! We hardly released it before the election, and it would have had more time in the spotlight had we released earlier than that. This does have some pros, like being ruthless in our prioritization, but that's a topic for another post.
```

Reply:
this is a cool feature. Github, I know you're listening, good job 👍